data_viz/viz.py

```diff
@@ -1,20 +1,46 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
+from tonami import pitch_process as pp
 
 # helper/utility functions specifically for plotting
 
+PITCH_FILEPATH = 'data/parsed/toneperfect_pitch_librosa_50-500-fminmax.json'
 
-def plot_tones(tones):
+# currently, filename specifications only work for the above PITCH_FILEPATH
+def plot_tones(filename='wo3_MV2_MP3.mp3'):
     """
-    Plots the list of tones.
+    Plots the specified tones.
 
     Args:
-        tones: a list/array of tones to be plotted.
+        filename: filename of tone to be plotted
```

Comment:
- plot_pitch_contour() might be a better function name -> more explicit, plot_tone is kind of ambiguous about what tone features we might be displaying
- by the same token, would be helpful to describe what kind of plot is being generated in the description